docs/dev/utils.rst

```diff
@@ -6,7 +6,7 @@ The ESP website has a bunch of other things in it that are useful to know.  Here
 Rendering and Templates
 -----------------------
 * We use a modified ``esp.utils.web.render_to_response`` instead of django's; it takes the template path, the request, and the context, and inserts useful things into the context, like information needed to render navbars, and the request.
-* To render a block of text that will be editable by admins, put a ``{% load render_qsd %}`` somewhere, and then put a ``{% render_inline_qsd name %}`` or ``{% render_inline_program_qsd program name %}`` in where you want the block.  If you want it to have default text, put it between the tags ``{% inline_qsd_block name %}`` and ``{% end_inline_qsd_block %}`` or ``{% inline_program_qsd_block program name %}`` and ``{% end_inline_program_qsd_block %}``.
+* To render a block of text that will be editable by admins, put a ``{% load render_qsd %}`` somewhere, and then put a ``{% render_inline_qsd ''name'' %}`` or ``{% inline_program_qsd_block prog ''name'' %}`` in where you want the block.  If you want it to have default text, put it between the tags ``{% inline_qsd_block ''name'' %}`` and ``{% end_inline_qsd_block %}`` or ``{% inline_program_qsd_block prog ''name'' %}`` and ``{% end_inline_program_qsd_block %}``.
```

Comment:
Wait, what are these changes?

Reply:
When I followed the instructions in this file as they were, it didn't work because it didn't recognize `render_inline_program_qsd`. I copied it from blocks that rendered successfully on other pages of the site. Do these changes seem wrong?

Reply:
I think these are both valid tags, which do slightly different things -- that's why the next sentence mentions the `{% inline_program_qsd_block %}`.  If one didn't work I'm guessing you may not have had the right template tags loaded, but if you search you can see lots of calls to both so they must be working somehow!

I would also prefer to keep "program" rather than "prog"; the usage depends on the template in question but "program" is clearer so we should prefer it.  Putting quotes around "name" is probably a good idea, but please make them actual double-quotes (`"`), not doubled single quotes (`''`).

Reply:
What is the difference between the two tags? I'm not sure I understand the second part of what you said. Why wouldn't I have had the right tags loaded/how would I know?

I didn't abbreviate it -- I thought that "prog" was literally the text that you have to write. Did I get that totally wrong?

Reply:
The difference is whether it has default text -- that's what the second sentence in this bullet refers to.  `render_*` is the one without default text; `*_block` has default text.  (No, the names don't make sense.)

For `prog`, it depends on the template you are writing.  It's whatever the program variable was in the template context.  It looks like that's `program` maybe 75% of the time, and `prog` maybe 25% of the time.  To find out which, you have to go look at where the template gets rendered, and whether its context had a `'prog'` or a `'program'`.